firebase-vertexai/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Unreleased
 * [changed] Added new exception type for quota exceeded scenarios.
 * [feature] `CountTokenRequest` now includes `GenerationConfig` from the model.
-
+* [changed] **Breaking Change**: `ImagenInlineImage.data` now returns the raw
+  image bytes (in JPEG or PNG format, specified in `ImagenInlineImage.mimeType`)
```

Comment:
nit:
```suggestion
  image bytes (in JPEG or PNG format, as specified in `ImagenInlineImage.mimeType`)
```

Reply:
Thanks! Updated and rewrapped to 80 chars.